faculty_cli/cli.py

```diff
@@ -177,29 +177,45 @@ def _check_credentials():
         _check_creds_file_perms()
 
 
+def _get_authenticated_user_id():
+    client = faculty.client("account")
+    return client.authenticated_user_id()
+
+
+def _list_projects():
+    """List all projects accessible by user."""
+    _check_credentials()
```

Comment:
`_check_credentials` isn't specific to `_list_projects` - it checks whether Faculty credentials are configured and available for use by any service client. This is not actually needed for the code to function, but instead displays a more user-friendly error message in the case that credentials are not configured. I think it makes sense to leave is as the first call in the CLI function itself rather than embedding it in this `_list_projects` function.